docs/permission/overview.md

```diff
@@ -0,0 +1,41 @@
+---
+id: overview
+title: Permission Framework Overview
+description: A high level overview of the Backstage permission framework
+---
+
+[The previous section](../auth/index.md) covered the various _authentication_ methods of Backstage, but Backstage can also _authorize_ specific data, APIs, or interface actions - meaning that Backstage has the ability to enforce rules about what actions are allowed for a given user of a system.
+
+By default, Backstage endpoints are not protected and all actions are available to anyone. However, configuring which users can access which resources and actions is a common need for many organizations. The permission framework allows integrators to achieve this through the use of granular permissioning for those resources and actions.
+
+The permission framework was designed with a few key properties in mind:
+
+- Flexibility: the framework allows integrators to configure many different authorization methods. This could include implementations like role-based access control (RBAC), attribute-based access control (ABAC), bespoke logic expressed in code, or integrations with external authorization providers.
+
+- Usability: the permission framework allows integrators to focus on configuring what they care about (the permission policy) by providing all of its moving parts out of the box. It also allows plugin authors to integrate support for permissions in their plugins without having to make any changes in Backstage core.
+
+## How does it work?
+
+- **Plugin authors** can add permission support in their plugins by declaring which resources from their plugins can be placed behind authorization and what types of actions users may take upon those resources.
```

Comment:
IMO, we can pull back a bit on the amount of space we devote in these docs to supporting plugin authors. Since integrators vastly outnumber plugin authors, and are also likely to be less familiar with technical details of Backstage, it seems like there's an opportunity to pitch these docs mostly at integrators, and just leave some kind of footnote saying "looking how to integrate support for permissions in your plugin? check out the docs for plugin owners here [link]".

Reply:
Hmm, it does make sense to me to optimize for the audience that is most likely to be reading these docs. On the other hand, as someone (even if that "someone" is most likely an adopter) who has basically 0 knowledge of this framework,  I would like a high level understanding of how everything works. That's why I've provided these bullet points + the diagram. In that vein, I think this description of the plugin's high level role in the structure answers the question of "where do these permissions come from?" Happy to continue iterating!

Reply:
Yeah, I can see that side of it too!

Reply:
What about moving the plugin author docs under the "Plugins" nav item? I see that we have "integrating with the software catalog" there, so "adding plugin permissions" feels like a natural step in that flow

Reply:
Yeah sounds good! Just need to make sure we aren't linking to these docs anywhere that might require a redirect if we move things around

Reply:
Yeah totally - but this PR is not targeting master right now, we've got an intermediate long-lived `permission-docs` branch for hacking on the docs together :)